docs/source/reference/modding/settings/input.rst

```diff
@@ -134,3 +134,76 @@ This setting has no effect on the camera rotation speed, which is controlled by
 camera sensitivity setting.
 
 This setting can only be configured by editing the settings configuration file.
+
+enable gyroscope
+----------------
+
+:Type:		boolean
+:Range:		True/False
+:Default:	False
+
+Enable support of camera rotation via gyroscope, supported by SDL2.
+Built-in gyroscopes on mobile devices are known to work.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro horizontal axis
+--------------------
+
+:Type:      string
+:Range:     x, y, z, -x, -y, -z
+:Default:   -x
+
+Maps an axis of gyroscope to the horizontal in-game axis. Minus sign inverses left and right directions.
+A setting is for the landscape mode, for other modes the game corrects data automatically.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro vertical axis
+------------------
+
+:Type:      string
+:Range:     x, y, z, -x, -y, -z
+:Default:   y
+
+Maps an axis of gyroscope to the vertical in-game axis. Minus sign inverses up and down directions.
+A setting is for the landscape mode, for other modes the game corrects data automatically.
```

Comment:
Like with horizontal axis, the used axis will be corrected automatically in the portrait mode.